### PR TITLE
TVM Overview - new guidance section

### PR DESCRIPTION
**Pull request by dansisson**

### CODE REVIEW

**Comment by dansisson** [comment]
If this PR needs approval, or if the commits need to be pulled into the private repo, contact @alkohli. Thanks.

**Comment by Jak-MS** [comment]
@alkohli 
- Can you review this PR?
- IMPORTANT: When this content is ready to merge, you must add `#sign-off` in a comment or the approval may get overlooked.


#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

File: azure-stack/hci/manage/trusted-launch-vm-overview.md [hunk 1]
```diff
@@ -34,6 +34,14 @@ Trusted launch is a security type that can be specified when creating Arc VMs on
 > [!NOTE]
 > VM guest boot integrity verification is not available.
 
+## Guidance
+
+- IgvmAgent is a component that is installed on all nodes in the Azure Stack HCI cluster. It enables support for isolated VMs such as Trusted launch Arc VMs for example.
+
+- As part of Trusted launch Arc VM creation, Hyper-V creates VM files on disk to store the VM state. By default, access to those VM files is restricted to host server administrators. Host administrators must ensure that the location where those VM files are stored remains appropriately access-restricted at all times.
+
+- VM live migration network traffic is not encrypted. We strongly recommend that you enable a network layer encryption technology such as SMB encryption or IPsec to protect live migration network traffic.
```
Comment (Karl-WE) [inline azure-stack/hci/manage/trusted-launch-vm-overview.md:43]: Please mind that SMB encryption will prevent use of RDMA. We need a remark box on that. IPsec seems like Overkill. Live mig traffic is in normal cases SMB 3,unless using old TCP compression.

**Review by Karl-WE — CHANGES_REQUESTED** [review]

**Comment by alkohli** [comment]
@Karl-WE Confirmed with Ram Jeyaraman that both of you have agreed on the current guidance. Signing off this PR.

#sign-off

**Comment by Court72** [comment]
Closing per https://github.com/MicrosoftDocs/azure-stack-docs/pull/3136#issuecomment-2197278500

**Pull request closed without merging**